### ISSUE TITLE
option for output file name (index.html)

### ISSUE THREAD

**Issue by stephenturner** [ropensci/postdoc#7]
Currently postdoc allows specification of the `outdir` when rendering package manuals. Ever considered allowing the user to specify the path of the resulting html instead of the output dir? Use case: instead of `./pkgname.html` I'd rather have `./index.html` so I can point github pages to the main branch of the repo.

**Comment by jeroen** [comment]
The thing is that the API is vectorized, so that would make it a bit messy to specify names. Now you can do e.g:

```r
postdoc::render_package_manual(c("MASS", "curl", "jsonlite"))
```

**Comment by stephenturner** [comment]
Yeah. I started poking around the code after filing the issue, ran into the same thing. I could see if length(package)==1 you could do this more easily but makes for an awkward API or else peculiar args.
